wdfn-server/waterdata/views.py

```diff
@@ -170,6 +161,26 @@ def return_404(*args, **kwargs):
     return abort(404)
 
 
+def get_cooperator_data(location_with_values, site_no):
+    """
+    Gets the cooperator data from a json file, currently a feature toggle, and limited to district codes 20 and 51
+
+    :param site_no: USGS site number
+    :param location_with_values: monitoring location data including district code
```

Comment:
Could we just pass in district_cd, rather than the whole location?

Reply:
Yep. And will do.